client/src/components/Verejne/Map/ClusterMarker/index.js

```diff
@@ -57,7 +64,7 @@ const ClusterMarker = ({
 
 export default compose(
   connect(
-    (state) => ({
+    state => ({
```

Comment:
I'm pretty sure that vacuumlabs eslint enforces these parentheses. Are you using eslint rules to format your code?